packages/fetchai/connections/p2p_libp2p/connection.py

```diff
@@ -62,7 +62,10 @@
 
 
 async def _golang_module_build_async(
-    path: str, log_file_desc: IO[str], loop: Optional[asyncio.AbstractEventLoop] = None
+    path: str,
+    log_file_desc: IO[str],
+    loop: Optional[asyncio.AbstractEventLoop] = None,
+    timeout: Optional[float] = LIBP2P_NODE_DEPS_DOWNLOAD_TIMEOUT,
```

Comment:
I think it should be `timeout: float = ...` unless you want to allow for it to be `None`

Reply:
correct